inventory.yml.sample

```diff
@@ -12,8 +12,8 @@ all:
     cluster_name: clustername
     base_dns_domain: example.lab
 
-    # OpenShift version (4.6.16, 4.7.33, 4.8.14 or 4.9.0)
-    openshift_full_version: 4.6.16
+    # Openshift version (4.6, 4.7, 4.8 or 4.9)
+    openshift_version: 4.8
```

Comment:
Would it be possible to make `openshift_version` support both `x.y` and `x.y.z` patterns? 

**Example:**
- User provides a `x.y` version. The `openshift_full_version_mapping` is used to lookup the supported `x.y.z` version.
- User provides a `x.y.z` version. That version is used as `openshift_full_version`.

I think this could significantly simplify our input.

With the currently proposed implementation, do I have to override `openshift_full_version` in order to get a specific OpenShift version? 
If so, what would happen if I set the vars in my inventory to:
```yaml
openshift_version: 4.8
openshift_full_version: 4.9.0
```

Reply:
Yes you would have to update the mapping. 
Err in terms of mismatched versions that probably wouldn't end well. We add a validation to check that doesn't happen